### PR TITLE
find-dependencies will run on private repos

### PR DESCRIPTION
**Pull request by jochman**
find-dependencies will merge the dependencies from local and content repo.

fixes: https://github.com/demisto/etc/issues/33848

### CODE REVIEW

**Comment by jochman** [comment]
image
image

**Comment by jochman** [comment]
@idovandijk fyi

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -912,11 +912,9 @@ def id_set_command(**kwargs):
 
 
 # ====================== merge-id-sets ====================== #
-@main.command(name='merge-id-sets',
-              hidden=True,
-              short_help='Merge two id_sets')
-@click.help_option(
-    '-h', '--help'
```
Comment (roysagi) [inline demisto_sdk/__main__.py:919]: why remove the help options?

**Review by roysagi — COMMENTED** [review]
This does seem good but since this is the find-dependencies command here we should run a content build (regular, nightly, and upload flow if possible) to check this change does not affect the current flows.

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (jochman) [reply demisto_sdk/__main__.py:919]: We don't need to specify it. `--help` will always be there, even without it.
The help option will only enable `-h`, an option that is not needed.

**Comment by jochman** [comment]
this pr has been scrambled.
https://github.com/demisto/demisto-sdk/pull/1264

**Pull request closed without merging**